openassessment/fileupload/views.py

```diff
@@ -0,0 +1,163 @@
+import hashlib
```

Comment:
Actually, the views listed here all refer to the filesystem backend. The S3 backend does not require any views. I agree that can create some confusion. I will move the views.py module to views_filesystem.py for clarity.